### PR TITLE
317 remove excess header values in VCF extract

### PR DESCRIPTION
**Pull request by RoriCremer**
Before:
image

After:
image

### CODE REVIEW

File: .dockstore.yml
Comment (gbggrant) [inline .dockstore.yml:120]: Do we get rid of these before we merge the PR?

File: scripts/variantstore/wdl/GvsExtractCallset.wdl
Comment (gbggrant) [inline scripts/variantstore/wdl/GvsExtractCallset.wdl:18]: Should these directory have the jira ticket number embedded (to see if the work has been merged into ah_var_store and so we can assume the directory and jar is stale / unneeded).

**Review by gbggrant — APPROVED** [review]
Looks good.

**Comment by gbggrant** [comment]
Ran this jar on the MSSNG call set and compared one of the output VCFs with that generated using the original (default) jar - differences found were as expected:

```
wm944-c33:gatk ggrant$ diff mssng_6_new_updated_header.vcf mssng_6_original.vcf 
1a2
> ##FILTER=<ID=EXCESS_ALLELES,Description="Site has an excess of alternate alleles based on the input threshold (default is 6)">
3a5
> ##FILTER=<ID=NAY,Description="Considered a NAY in the Yay, Nay, Grey table">
17a20,21
> ##INFO=<ID=END,Number=1,Type=Integer,Description="Stop position of the interval">
> ##INFO=<ID=ExcessHet,Number=1,Type=Float,Description="Phred-scaled p-value for exact test of excess heterozygosity">
```

File: scripts/variantstore/wdl/GvsExtractCallset.wdl
Comment (mcovarr) [reply scripts/variantstore/wdl/GvsExtractCallset.wdl:18]: Yeah I agree that would be good. The combo of embedding the ticket number in the branch name and the "copy your custom GATK jar" script that Kristian shared the other day we seem to get that.

**Review by mcovarr — APPROVED** [review]
LGTM once Travis is happy. 👍

File: scripts/variantstore/wdl/GvsExtractCallset.wdl
Comment (RoriCremer) [reply scripts/variantstore/wdl/GvsExtractCallset.wdl:18]: I like this! Maybe post-Charlie we can consolidate a set of rules for jars? It's certainly too haphazard now

File: .dockstore.yml
Comment (RoriCremer) [reply .dockstore.yml:120]: Ideally yes, but with tests taking so long, sometimes we dont. It would be great to do a full clean up of all the extra branches in dockstore too

**Pull request merged**
